ush/forecast_postdet.sh

```diff
@@ -736,11 +736,11 @@ MOM6_postdet() {
   [[ ! -d ${COM_OCEAN_HISTORY} ]] && mkdir -p "${COM_OCEAN_HISTORY}"
 
   # Link output files
-  if [[ "${RUN}" =~ "gfs" ]]; then
+  if [[ "${RUN}" =~ "gfs" || "${RUN}" =~ "gefs" ]]; then
     # Link output files for RUN = gfs
 
     # TODO: get requirements on what files need to be written out and what these dates here are and what they mean
-    export ENSMEM=${ENSMEM:-01}
+    export MEMNUM=${MEMNUM:-00}
```

Comment:
elevate this to `function common_predet()` in `forecast_predet.sh` and remove from `function CICE_postdet()`
Also note that it is 2 digits here and 3 digits in `CICE_postdet` line 923.

Reply:
Done (and three digits).